### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by megahypercat** [sqlitebrowser/sqlitebrowser#3133]
### What did you do?

Open large table

### What did you expect to see?

It loads in a couple seconds, since a SQlite query on the same data takes a second for the whole table, and a couple MS for just the queried data.

### What did you see instead?

Opening big table is slow, even filtering for queries that only have a couple of matches takes forever - They don't manage to complete a lot of the time.

Performance is good with normal SQlite.
Performance is good with small tables in the same file.
Performance is good with SQlite studio.
The suggestions of setting prefetch block size and symbol limit did not affect performance.

DB size: 155MB
rows (in the big table): 524000
OS: Windows 10

### DB4S Version

3.12.2

### What OS are you seeing the problem on?

Windows

### OS version

Enterprise LTSC 

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by chrisjlocke** [comment]
> The suggestions of setting prefetch block size

DB4S has to do two things.  Work out how many records in the table, then display X records.  The number in X is determined by the prefetch block size.  It'll either return 500,000 records, or 10.
You've not really given much information about the data in the table.  How many rows are you displaying?  Are there two columns or 50,000?  Do these columns contains integers and small text fields or tons of text?
I've not got your database, but just created a database of 1.6 million rows with no problems.  

![image](https://user-images.githubusercontent.com/7872218/184454879-072ece86-99ed-4814-a394-66701c0f2d91.png)

Changing a filter takes less than a second.  That's with a prefetch size of 50,000.

Obviously if I increase the cell contents with 1 MB of data in each field, things slow a bit more.  It has to fill each cell with data.

We need a bit more information on your data you're displaying.

**Comment by sky5walk** [comment]
Please edit the title. 
What is the bug?

**Comment by megahypercat** [comment]
I have found that this is due to using it over a network share.
I'm guessing that this is not considered supported?

Works ok with SQlite studio so it's a bit unfortunate, I want to use DB4S.

**Comment by sky5walk** [comment]
Yes, you must choose your [tools and environment appropriately](https://www.sqlite.org/draft/useovernet.html).

**Comment by chrisjlocke** [comment]
155 MB isn't that large to throw over a network.  Is it a slow Wi-Fi network?
